easybuild/easyconfigs/m/MAGeCK-VISPR/MAGeCK-VISPR-0.5.6-foss-2023a.eb

```diff
@@ -0,0 +1,74 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>; Guillermo AR Piccoli <guillermo.piccoli@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'PythonBundle'
+
+name = 'MAGeCK-VISPR'
+version = '0.5.6'
+
+homepage = 'https://bitbucket.org/liulab/mageck-vispr/'
+description = """MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9
+  screens The workflow combines the MAGeCK algorithm to identify essential genes from CRISPR/Cas9 screens considering
+  multiple conditions with VISPR to interactively explore results and quality control in a web-based frontend."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('SciPy-bundle', '2023.07'),
+    ('Meson', '1.4.0')
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('PyYAML', '5.4.1', {
+        'modulename': 'yaml',
+        'checksums': ['607774cbba28732bfa802b54baa7484215f530991055bb562efbed5b2f20a45e'],
+    }),
+    ('pyproject-metadata', '0.7.1', {
+        'checksums': ['0a94f18b108b9b21f3a26a3d541f056c34edcb17dc872a144a15618fed7aef67'],
+    }),
+    ('setuptools-scm', '8.0.4', {
+        'modulename': 'setuptools_scm',
+        'checksums': ['b5f43ff6800669595193fd09891564ee9d1d7dcb196cab4b2506d53a2e1c95c7'],
+    }),
+    ('calver', '2022.6.26', {
+        'checksums': ['e05493a3b17517ef1748fbe610da11f10485faa7c416b9d33fd4a52d74894f8b'],
+    }),
```

Comment:
I struggle to see where this requirements come from.. In fact. there doesn't seem to be much of *any* overlap between all these extensions. 
MAGeCK-VISPR lists 
fastqc >=0.11.4
mageck >=0.5.9
vispr >=0.4.11
snakemake >=3.6.0
cutadapt >=1.9.1
jinja2 >=2.8
which really has nothing in common with anything here. 

In addition, this software has basically been abandoned for some 3 years now, so the idea that it would have requirements on newer versions of anything does not seem reasonable to me. 


Edit: I just tested this now, removed Meson, removed all these extensions, added PyYAML to the dependencies, and it builds fine like this.
I looked through the source code and it needs
1. pandas (provided by SciPy-bundle)
2. jinja2 (also happens to be provided by SciPy-bundle)
3. yaml (provided by PyYaml)
No other extensions are needed here at all.